lean4-infoview/src/infoview/rpcSessions.tsx

```diff
@@ -146,6 +146,7 @@ export class RpcSessions implements Disposable {
     async call<T>(pos: DocumentPosition, method: string, params: any): Promise<T | undefined> {
         const sesh = await this.sessionAt(pos.uri)
         if (!sesh) {
+            console.log(`call ${method} dropped because session does not yet exist`)
```

Comment:
We may not want this log in production, my guess is it might show up a lot and spam the console.

Reply:
These console.log messages only show up in the console of the debugger we use while developing the extension.  For example, the debugger in the background has the console.log messages but the "user's vs code" instance shows nothing:

![image](https://user-images.githubusercontent.com/18707114/175186766-694527c1-4193-4259-b607-9ee00836a0ca.png)